app/javascript/components/student/solution-summary/MentoringComboButton.tsx

```diff
@@ -49,11 +47,11 @@ export const MentoringComboButton = ({
   return (
     /* TODO: Extract into a common component in the future */
     <div className={classNames.join(' ')}>
-      {hasMentorDiscussionInProgress && links.inProgressDiscussion ? (
+      {mentoringStatus == 'in_progress' && links.inProgressDiscussion ? (
```

Comment:
A previous file used `===` instead of `==`, which is a good idea I think.

Reply:
(Bump @kntsoriano)